src/bin/miri.rs

```diff
@@ -6,14 +6,13 @@ extern crate rustc_driver;
 
 use miri::interpreter;
 use rustc::session::Session;
-use rustc_driver::{driver, CompilerCalls, Compilation};
+use rustc_driver::{driver, CompilerCalls};
 
 struct MiriCompilerCalls;
 
 impl<'a> CompilerCalls<'a> for MiriCompilerCalls {
     fn build_controller(&mut self, _: &Session) -> driver::CompileController<'a> {
         let mut control = driver::CompileController::basic();
-        control.after_analysis.stop = Compilation::Stop;
```

Comment:
Ah, good point. I guess technically it doesn't make sense for any Miri tests to be run-pass because the execution all happens at compile-time, but this is fine for now.

Later we can rewrite those tests in a smarter way.